app/my-bets/page.tsx

```diff
@@ -36,40 +55,72 @@ export default function MyBetsPage() {
-                <div className="bg-surface-surface-0 text-2xs border border-outline-low-em rounded-6 p-1 px-1.5 ml-2">
-                  {data?.userPositions.length ?? "-"}
-                </div>
+                <TabBetCounter>{userPositions?.length ?? "-"}</TabBetCounter>
+              </TabStyled>
+              <TabStyled>
+                Active
+                <TabBetCounter>
+                  {filterActiveBets(userPositions).length ?? "-"}
+                </TabBetCounter>
+              </TabStyled>
+              <TabStyled>
+                Unredeemed
+                <TabBetCounter>0</TabBetCounter>
+              </TabStyled>
+              <TabStyled>
+                Complete
+                <TabBetCounter>
+                  {filterClompleteBets(userPositions).length ?? "-"}
+                </TabBetCounter>
               </TabStyled>
-              <TabStyled>Active</TabStyled>
-              <TabStyled>Unredeemed</TabStyled>
-              <TabStyled>Complete</TabStyled>
             </TabHeader>
 
             <TabBody className="mt-8">
               <TabPanel className="space-y-4">
                 {isLoading ? (
                   <LoadingBets />
+                ) : emptyData ? (
+                  <EmptyState />
                 ) : (
-                  data &&
-                  data?.userPositions.map((position: UserPosition) => (
+                  userPositions.map((position: UserPosition) => (
                     <CardBet userPosition={position} key={position.id} />
                   ))
                 )}
               </TabPanel>
               <TabPanel>
-                <div className="bg-surface-primary-accent-1 p-5 rounded-4">
-                  in development..
-                </div>
+                {isLoading ? (
+                  <LoadingBets />
+                ) : filterClompleteBets.length === 0 ? (
+                  <div className="bg-surface-surface-1 p-6 rounded-12 space-y-4">
+                    <p>No active bets</p>
+                  </div>
+                ) : (
+                  filterActiveBets(userPositions).map(
+                    (position: UserPosition) => (
+                      <CardBet userPosition={position} key={position.id} />
+                    )
+                  )
+                )}
               </TabPanel>
               <TabPanel>
-                <div className="bg-surface-danger-accent-1 p-5 rounded-4">
-                  in development..
+                <div className="bg-surface-surface-1 p-6 rounded-12 space-y-4">
+                  <p>No redeemable bets</p>
                 </div>
               </TabPanel>
-              <TabPanel>
-                <div className="bg-surface-warning-accent-1 p-5 rounded-4">
-                  Come back in a week!
-                </div>
+              <TabPanel className="space-y-4">
+                {isLoading ? (
+                  <LoadingBets />
+                ) : filterClompleteBets.length === 0 ? (
```

Comment:
If `filterClompleteBets` becomes an array this is correct. Otherwise this must change.